tests/test_swift.py

```diff
@@ -104,17 +108,23 @@ def test_destroy_app(self):
             for channel in info.channel:
                 self.assertNotIn(name, self.swift._subscribers[channel])
 
-    def test_update_frames(self):
+    def test_update_frames_inclusive(self):
+        orgFramesSet = {dockWidget.widget() for dockWidget in self.swift._dockWidgets["app1"]}
+        newFramesSet = orgFramesSet | {QWidget()}
+        self.swift._apps["app1"].frames.return_value = tuple(newFramesSet)
+        self.swift.updateFrames("app1")
+        finalFramesSet = {dockWidget.widget() for dockWidget in self.swift._dockWidgets["app1"]}
+        self.assertFalse(finalFramesSet & (orgFramesSet - newFramesSet))
```

Comment:
By definition `orgFramesSet - newFramesSet` is empty.
I think this assertion should be done for general cases (not inclusive nor exclusive).

Reply:
Oh, you are right! I will remove the line.